src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -623,7 +623,7 @@ fluent-data-grid-cell.no-ellipsis {
 }
 
 .filter-input-container {
-    width: 75%;
+    width: 85%;
```

Comment:
was this just for aesthetics?

Reply:
Some values are long, and there is now a count. Increase is to show more of the value in the UI.